eqcorrscan/core/lag_calc.py

```diff
@@ -190,7 +190,7 @@ def _concatenate_and_correlate(streams, template, cores):
 def xcorr_pick_family(family, stream, shift_len=0.2, min_cc=0.4,
                       horizontal_chans=['E', 'N', '1', '2'],
                       vertical_chans=['Z'], cores=1, interpolate=False,
-                      plot=False, plotdir=None):
+                      plot=False, plotdir=None, npy=False, npydir=None):
```

Comment:
It would be good to match arguments with the args in #420 e.g.:
```suggestion
                      plot=False, plotdir=None, export_cc=False, cc_dir=None):
```